Cargo.toml

```diff
@@ -8,15 +8,15 @@ name = "bevy_magic_light_2d"
 crate-type = ["cdylib", "lib", "staticlib"]
 path = "src/lib.rs"
 
+[features]
+default = ["wayland"]
+wayland = ["bevy/wayland"]
+
```

Comment:
I don't think these features should be added here.

Bevy plugins should allow the game crate to pick which bevy features it enables, without plugins enabling any of them by default, except for the ones require for the plugin to work.

Reply:
I agree. Wayland should not be a feature